backend/packages/Upgrade/src/api/services/ErrorService.ts

```diff
@@ -23,8 +22,11 @@ export class ErrorService {
     return this.errorRepository.paginatedFind(limit, offset, filter);
   }
 
-  public create(error: ExperimentError): Promise<ExperimentError> {
-    this.log.info(`Inserting an error => ${JSON.stringify(error)}`);
+  public create(error: ExperimentError, logger: UpgradeLogger): Promise<ExperimentError> {
+    if( logger ) {
+      logger.addFromDetails(__filename, 'create');
+      logger.info({ stdout: 'Inserting an error', details: error });
+    }
     return this.errorRepository.save(error).catch((err) => {
```

Comment:
In case you are logging after this method you need to add `logger.addFromDetails` again so as to update the from detail

Reply:
Ok, removing addFromDetails as discussed in the scrum meeting